examples/name-service/programs/name-service-without-macros/src/lib.rs

```diff
@@ -154,41 +181,11 @@ pub enum CustomError {
     Unauthorized,
 }
 
-#[light_system_accounts]
-#[derive(Accounts, LightTraits)]
-pub struct CreateRecord<'info> {
-    #[account(mut)]
-    #[fee_payer]
-    pub signer: Signer<'info>,
```

Comment:
feels intuitive to me to keep the signer here so that it can be used for anchor checks see comment for `LightSystemAccounts`